src/configs/sys_config.py

```diff
@@ -0,0 +1,23 @@
+
+# System Configuration
+
+system_config = {
+    "num_users": 4,
+    "experiment_path": "./experiments/",
+    "dset": "cifar10",
+    "dump_dir": "./expt_dump/",
+    "dpath": "/data/unagi0/anakewat/imgs/",
+    "device_ids": {
+        "iid_dispfl": {"node_0": [2], "node_1": [3], "node_2": [4], "node_3": [5], "node_4": [2]},
```

Comment:
algorithm names shouldn't be present inside `device_ids` and system_config in general. The goal is to make the system config independent of algorithms.

Reply:
how do we want to pass this in then